src/connections.jl

```diff
@@ -12,37 +12,36 @@ const CONNECTION_OPTION_DEFAULTS = Dict{String, String}(
 function _connection_parameter_dict(;
     client_encoding::String="UTF8",
     application_name::String="LibPQ.jl",
-    connection_options::Dict{String, String}=Dict{String, String}(),
+    connection_options::Dict{String,String}=Dict{String,String}(),
 )
     keep_option((k, v)) = !(k == "TimeZone" && v == "")
 
-    Dict{String, String}(
+    return Dict{String,String}(
         "client_encoding" => client_encoding,
         "application_name" => application_name,
-        "options" => join(
-            imap(Iterators.filter(keep_option, connection_options)) do (k, v)
-                "-c $k=$(show_option(v))"
-            end,
-            " ",
-        ),
+        "options" => join(imap(Iterators.filter(keep_option, connection_options)) do (k, v)
+            "-c $k=$(show_option(v))"
+        end, " "),
     )
 end
 
-const CONNECTION_PARAMETER_DEFAULTS = _connection_parameter_dict(
+const CONNECTION_PARAMETER_DEFAULTS = _connection_parameter_dict(;
     connection_options=CONNECTION_OPTION_DEFAULTS
 )
 
 # https://www.postgresql.org/docs/10/libpq-connect.html#LIBPQ-PQCONNECTSTARTPARAMS
-const CONNECTION_STATUS_MESSAGES = Dict{libpq_c.ConnStatusType, String}(
+const CONNECTION_STATUS_MESSAGES = Dict{libpq_c.ConnStatusType,String}(
     libpq_c.CONNECTION_OK => "Connection is ready",
     libpq_c.CONNECTION_BAD => "Connection failed",
     libpq_c.CONNECTION_STARTED => "Waiting for connection to be made",
     libpq_c.CONNECTION_MADE => "Connection OK; waiting to send",
     libpq_c.CONNECTION_AWAITING_RESPONSE => "Waiting for a response from the server",
-    libpq_c.CONNECTION_AUTH_OK => "Received authentication; waiting for backend start-up to finish",
+    libpq_c.CONNECTION_AUTH_OK =>
+        "Received authentication; waiting for backend start-up to finish",
     libpq_c.CONNECTION_SETENV => "Negotiating environment-driven parameter settings",
     libpq_c.CONNECTION_SSL_STARTUP => "Negotiating SSL encryption",
-    libpq_c.CONNECTION_CHECK_WRITABLE => "Checking if connection is able to handle write transactions",
+    libpq_c.CONNECTION_CHECK_WRITABLE =>
+        "Checking if connection is able to handle write transactions",
```

Comment:
i wonder if this case further motivates https://github.com/invenia/BlueStyle/issues/59 ?